pkg/context/context.go

```diff
@@ -444,3 +445,19 @@ func (c *Context) SetSensitiveValues(vals []string) {
 		c.Err = err
 	}
 }
+
+// UserAgent returns a string that can be used as a user agent for porter.
+func (c *Context) UserAgent() string {
```

Comment:
What do you think about making this a function in pkg/version.go? Nothing about this relies on the context, so this moves the function closer to the data it depends upon.